cfme/infrastructure/config_management.py

```diff
@@ -520,3 +440,142 @@ def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None):
         self.ssl = ssl
         self.credentials = credentials
         self.key = key or name
+
+
+@navigator.register(ConfigManager, 'All')
+class MgrAll(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Configuration', 'Configuration Management')(None)
+
+    def resetter(self):
+        accordion.tree('Providers',
+                       version.pick({version.LOWEST: 'All Red Hat Satellite Providers',
+                                     '5.6': 'All Configuration Manager Providers'}))
+
+        tb.select('Grid View')
+
+    def am_i_here(self):
+        return summary_title() == 'All Configuration Management Providers'
+
+
+@navigator.register(ConfigManager, 'Add')
+class Add(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        cfg_btn('Add a new Provider')
+
+
+@navigator.register(ConfigManager, 'Edit')
+class Edit(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Edit Selected item')
+
+
+@navigator.register(ConfigManager, 'RefreshRelationships')
+class RefreshRelationships(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Refresh Relationships and Power states', invokes_alert=True)
+
+
+@navigator.register(ConfigManager, 'Remove')
+class Remove(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Remove selected items from the VMDB', invokes_alert=True)
```

Comment:
As stated above, we don't want navigation destinations for this and the RefreshRelationships.